app/views/posts/index.html.erb

```diff
@@ -1,6 +1,37 @@
-<h1>Welcome to post index page</h1>
-<ul>
-  <% @posts.each do |post| %>
-    <li><%= post.title %></li>
-  <% end %>
-</ul>
+<div class='user_show_container'> 
+  <div class='headline_container'>
+    <div class="image_container" style="width: 15rem; height: 15rem;">
+      <% if @user.photo %>
+        <img src="<%= @user.photo %>" class="" >
+      <% else %>
+        <p> No image found </p>
+      <% end %>
+    </div>
+    <div class="content_show_container">
+      <h1 class="">User Name: <%= @user.name %></h1>  
+      <h6 class="">Number of posts: <span><%= @user.posts_counter %></span></h6>
+    </div>
+  </div>
+
+  <div class="user_posts">
+    <h2>Posts</h2>
+    <% @user.recent_posts.each do |post| %>
+      <div class='posts_list'>
+        <h3>Title: <%= post.title %></h3>
+        <%= post.text %> <br> <br>
+        Comments: <%= post.comments_counter %>
+        Likes: <%= post.likes_counter %>
+      </div>
+      <% post.comments.each do |comment| %>
+        <div class='posts_list'>
+          <h3><%= comment.text %>: <%= comment.text %></h3>
+        </div>
+      <% end %>
+      <% if post == @user.recent_posts.last %>
+      <div class='user_show_button'>
+        <%= link_to "Pagination", user_post_path(@user, post),class: "btn_show_posts" %>
+        </div>
```

Comment:
- [ ] Kindly consider adding the name of the user who wrote the comment on the display as per the given design.
given design
<img width="384" alt="Screenshot 2023-07-28 at 10 43 34 PM" src="https://github.com/ridabensalem/blog-app/assets/96683843/73d49561-09cd-4d77-8976-654481a25409">

your design
<img width="662" alt="Screenshot 2023-07-28 at 10 55 22 PM" src="https://github.com/ridabensalem/blog-app/assets/96683843/c45c3b44-2791-4bd0-96a7-3d23bbef6c69">